src/matter/cluster/LevelControlCluster.ts

```diff
@@ -55,23 +55,23 @@ const MoveToClosestFrequencyCommandRequest = TlvObject({
 /** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.4 */
 const features = {
     /** Dependency with the On/Off cluster */
-    onOffClusterDependency: BitFlag(0), // default: true
+    onOff: BitFlag(0), // default: true
 
     /** Behavior that supports lighting applications. */
     lighting: BitFlag(1),
 
     /** Supports frequency attributes and behavior. */
-    supportFrequency: BitFlag(2)
+    frequency: BitFlag(2)
 };
 
 /** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.6.2.1 */
-const enum MoveMode {
+export const enum MoveMode {
```

Comment:
Needed if we want to use it to send or code with ... so experted it now because needed in my local version. I could revert here but I assume that it do not make that big of a difference